### ISSUE TITLE
It breaks accounts!!!!11

### ISSUE THREAD

**Issue by AidenMcCue** [rxzyx/prodigy-hack#3]
I did it now everytime i try to battle it stays stuck on the loadig screen

**Comment by rxzyx** [comment]
Try setting everything back to normal. Reply what happens.

**Comment by AidenMcCue** [comment]
How do I set everything back to normal? Please and Thank you.

**Comment by AidenMcCue** [comment]
I used the damage and it was fine but as soon as i used Invincibility that happend.

**Comment by rxzyx** [comment]
Try re-using the hacks, eg level and set it to something random like 69 or whatever. I'm working to fix this now.

**Comment by AidenMcCue** [comment]
ok thnk you ill try that

**Comment by rxzyx** [comment]
fyi: Invincibility can't be the problem since it's client side.

**Comment by AidenMcCue** [comment]
Would a video recording of it help you fix it btw?

**Comment by rxzyx** [comment]
Nope. I'll let you know once I'm done.

**Comment by rxzyx** [comment]
I fixed it! The pets hack is bugged!

**Comment by AidenMcCue** [comment]
> fyi: Invincibility can't be the problem since it's client side.

then i dont know why it did it

**Comment by rxzyx** [comment]
I will now fix the pets hack since thats the problem.

**Comment by rxzyx** [comment]
Once you will use the fixed pets hack, your progress should be restored back to normal.

**Comment by rxzyx** [comment]
I'll leave a comment once I re-coded the pets hack.

**Comment by AidenMcCue** [comment]
ok thank you so so so so so much! Btw I starred this and Im now following You

**Comment by rxzyx** [comment]
Thanks!

**Comment by AidenMcCue** [comment]
![Screenshot 2022-04-24 2 42 37 PM](https://user-images.githubusercontent.com/104032778/164992551-3e4ac081-a520-4c8d-978a-53c48921f9ec.png)

**Comment by AidenMcCue** [comment]
Sorry I accidently closed this issue i unclosed it tho

**Comment by AidenMcCue** [comment]
It dont work still i just used the pets and whenever i do a battle my game freezes

**Comment by rxzyx** [comment]
I didn't update it yet. For me my private code has a feature that resets all my pets, and that works. But I'm going to try another work-around.

**Comment by rxzyx** [comment]
Fix is like 80% finished

**Comment by rxzyx** [comment]
99% finished literally

**Comment by rxzyx** [comment]
@AidenMcCue DONE!!! 
https://github.com/rxzyx/prodigy-hack/blob/main/unlockPets.js
RUN THE CODE ABOVE AND IT NOW YOU CAN PLAY MATCHES!

**Comment by AidenMcCue** [comment]
ok thank you let me try again

**Comment by AidenMcCue** [comment]
Omg thank you, you are a lifesaver!

**Comment by rxzyx** [comment]
No problem!
